### ISSUE TITLE
Checksum error with Julia 1.3.0

### ISSUE THREAD

**Issue by ghost** [JuliaLang/julia#34047]
If you test older version, everything is ok:

~~~
C:\Program Files\7-Zip> 7z t C:\Users\Steven\Desktop\julia-1.2.0-win64.exe

7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 58893624 bytes (57 MiB)

Testing archive: C:\Users\Steven\Desktop\julia-1.2.0-win64.exe
--
~~~

but if you check newer version, it returns an error:

~~~
C:\Program Files\7-Zip> 7z t C:\Users\Steven\Desktop\julia-1.3.0-win64.exe

7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 60273568 bytes (58 MiB)

Testing archive: C:\Users\Steven\Desktop\julia-1.3.0-win64.exe

WARNINGS:
Checksum error
~~~

**Comment by ViralBShah** [comment]
Just to make sure, can you download 1.3 again and run the checksum?

**Comment by cup** [comment]
I tested this multiple times before posting, with multiple downloaders...

**Comment by tariksalay** [comment]
having the same error

**Comment by cup** [comment]
Workaround:

https://julialang.org/downloads/nightlies

**Comment by cup** [comment]
Still a problem with v1.3.1

**Comment by muv1** [comment]
This has been resolved as of 1.4.1 (maybe sooner), as `innoextract` can be used instead.
